<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="group1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>400</width>
      <height>430</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Your Picture</string>
    </property>
    <widget class="QWidget" name="widget1" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>28</y>
       <width>402</width>
       <height>402</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>289</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo_aiacademy.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>FACE SEARCH SYSTEM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="subTitle">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>730</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>A system for searching human face in database</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="group2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>400</width>
      <height>430</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Top Results</string>
    </property>
    <widget class="QWidget" name="widget2" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>28</y>
       <width>402</width>
       <height>402</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="selectBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 8px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none;
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Select Picture</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="group3">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>110</y>
      <width>150</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>      </string>
    </property>
    <widget class="QWidget" name="widget3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>28</y>
       <width>150</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>179</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget5" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>329</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>590</y>
      <width>500</width>
      <height>133</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QFrame" name="frame3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>33</y>
       <width>500</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:4px groove rgb(25, 25, 25);
border-radius: 20px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
    <widget class="QLabel" name="resultLb">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FontAwesome</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>They look DIFFERENT!</string>
     </property>
    </widget>
    <widget class="QLabel" name="resultLb1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FontAwesome</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Similarity score: </string>
     </property>
    </widget>
    <widget class="QLabel" name="scoreLb">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>110</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FontAwesome</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>90%</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="scoreLb2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>260</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>90%</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLb3">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>410</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>90%</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLb4">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>560</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>90%</string>
    </property>
   </widget>
   <zorder>subTitle</zorder>
   <zorder>group1</zorder>
   <zorder>logo</zorder>
   <zorder>title</zorder>
   <zorder>group2</zorder>
   <zorder>selectBtn</zorder>
   <zorder>group3</zorder>
   <zorder>group4</zorder>
   <zorder>scoreLb2</zorder>
   <zorder>scoreLb3</zorder>
   <zorder>scoreLb4</zorder>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
